test/src/Entity/FilterModel.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use PommProject\ModelManager\Model\Model;
+use PommProject\ModelManager\Model\Projection;
+use PommProject\ModelManager\Model\ModelTrait\WriteQueries;
+
+use PommProject\Foundation\Where;
+
+use AppBundle\Entity\AutoStructure\Filter as FilterStructure;
+use AppBundle\Entity\Filter;
+
+/**
+ * FilterModel
+ *
+ * Model class for table filter.
+ *
+ * @see Model
+ */
+class FilterModel extends Model
+{
+    use WriteQueries;
+
+    /**
+     * __construct()
+     *
+     * Model constructor
+     *
+     * @access public
+     */
+    public function __construct()
+    {
+        $this->structure = new FilterStructure;
+        $this->flexible_entity_class = '\AppBundle\Entity\Filter';
```

Comment:
Use `\AppBundle\Entity\Filter::class` instead, we are sure that the class exists.